### ISSUE TITLE
fetchIssuers question

### ISSUE THREAD

**Issue by lostcause** [markdegrootnl/omnipay-ideal#4]
Hello,

I am trying to use your package and I seem to have some issues when I want to fetch the list of issuers. I might be doing something wrong, as I couldn't find any information about how I should set up the request, and I also don't get any errors back so I could see why it's not working. This is the code I have so far:

`$gateway = Omnipay::create('Ideal');
        $gateway->setAcquirer('ing');
        $gateway->setMerchantId('0000000');
        $gateway->setSubId('0');
        $gateway->setPublicKeyPath(base_path().'/resources/ssl/cert.cer');
        $gateway->setPrivateKeyPath(base_path().'/resources/ssl/private.cer');
        $gateway->setPrivateKeyPassphrase('password');
        dd($gateway->fetchIssuers());`

and I get back a FetchIssuersRequest object, which among others, contains 
`#response: null`

Could you point me to an example of how to setup and use your package? Thank you!

**Comment by markdegrootnl** [comment]
Hi @lostcause, 

Did you register with an iDeal provider and configure the MerchantId, SubId, PublicKey, PrivateKey and PrivateKeyPassphrase? Even in demo mode you need this to work.

**Comment by lostcause** [comment]
Yes, I am using a customer's iNG account, and I configured all properties.
Is there a way to check the response from ideal for any errors?

P.S. Using the library that iDEAL provides (for vanilla PHP) as an example works fine.
